app/scraper/__init__.py

```diff
@@ -75,6 +76,9 @@ def scrape(caches_active, face_book_cookie, people_search_session_cookie, csrf_t
         people = departmental.integrate(people)
         cache.set(cache_key, people)
 
+    image_uploader = ImageUploader()
+    image_uploader.deleted_unused_imgs(people)
```

Comment:
Perhaps run this through `face_book` instead of creating a new instance of ImageUploader.

Reply:
I agree it seems a bit wasteful to create a new instance. I originally had it run at the end of `scrape(...)` in `face_book` but you had concerns about the image deletion occurring in the middle of the scraping, so this was my attempt to have it run "at the very end." Let me know if you have any suggestions on what makes the most sense!

Reply:
What I'm envisioning is that you could add a separate method to `face_book` that could be called in this place to handle the deletion.

Reply:
Ah good call!